starter_code/css/mobile.css

```diff
@@ -0,0 +1,752 @@
+	max-width: 100%;
+	height: auto;
+}
+
+.logo {
+  font-weight: 800;
+}
+
+header {
+  background: #033048;
+  overflow: hidden;
+}
+
+header .logo {
+  color: #fff;
+  float: left;
+}
+
+header nav {
+  float: right;
+}
+
+header nav ul li {
+  display: inline-block;
+}
+
+header nav ul li a {
+  color: #f9e42e;
+}
+
+#hero {
+  background: url(../images/header_bg.jpg) no-repeat top center #131e27;
+}
+
+#hero .logo {
+  color: #fff;
+}
+
+#hero h2 {
+  text-align: center;
+  color: #fff;
+  font-weight: 600;
+}
+
+.button {
+  font-weight: 800;
+  text-align: center;
+  display: block;
+  margin: 0 auto;
+}
+
+.yellow {
+  background: #f9e42e;
+  color: #033048;
+}
+
+.blue {
+  background: #033048;
+  color: #fff;
+}
+
+h3 {
+  color: #033048;
+  font-weight: 600;
+}
+
+#benefits ul {
+  list-style: disc;
+}
+
+#testimonial {
+  background: url(../images/testimonial_bg.jpg) no-repeat top center #071324;
+  color: #fff;
+}
+
+blockquote {
+  font-weight: 300;
+  font-style: italic;
+  text-align: center;
+  margin: 0 auto;
+}
+
+blockquote span {
+  font-weight: 600;
+  display: block;
+  font-style: normal;
+}
+
+#posts {
+  display: inline-block;
+  vertical-align: top;
+}
+
+#sidebar {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.post {
+  border-bottom: 1px solid #043049;
+}
+
+.post h2 {
+  font-weight: 600;
+}
+
+.post h2 a {
+  color: #043049;
+}
+
+.post p {
+  color: #606161;
+}
+
+.post p:first-of-type:first-letter {
+  float: left;
+  font-weight: 600;
+}
+
+.readmore {
+  text-align: right;
+}
+
+.readmore a {
+  color: #043049;
+  font-weight: 600;
+}
+
+#sidebar h3 {
+  font-weight: 800;
+  color: #043049;
+}
+
+#sidebar ul li a {
+  color: #043049;
+}
+
+#sidebar p a {
+  color: #043049;
+  font-weight: 600;
+}
+
+#bottom-cta {
+  background: #f9e42e;
+  text-align: center;
+}
+
+footer {
+  background: #121212;
+  text-align: center;
+}
+
+footer .logo {
+  color: #f9e42e;
+}
+
+footer ul li {
+  display: inline-block;
+  list-style: none;
+}
+
+footer p {
+  color: #fff;
+  font-weight: 600;
+}
+
+
+
+
+@media(min-width: 320px) {
+	/* small portrait screens */
+
+	/******************************************
+	/* BASE STYLES
+	/*******************************************/
+
+	body {
+	  font-size: 16px;
+	  line-height: 1.2;
+	}
+
+	/******************************************
+	/* LAYOUT
+	/*******************************************/
+
+	.logo {
+	  font-size: 36px;
+	}
+
+	header {
+	  padding: 15px;
+	}
+
+	header nav {
+	  margin-top: 5px;
+	}
+
+	header nav ul li {
+	  margin: 0 0 0 10px;
+	  font-size: 20px;
+	}
+
+	#hero {
+	  height: 500px;
+	  padding-top: 15px;
+	}
+
+	#hero h2 {
+	  font-size: 36px;
+	  margin: 75px 0 75px;
+	}
+
+	.button {
+	  border-radius: 5px;
+	  padding: 12px 0;
+	  width: 180px;
+	  font-size: 20px;
+	}
+
+	.blue {
+	  font-size: 20px;
+	}
+
+	#benefits {
+	  padding: 30px 15px;
+	}
+
+	h3 {
+	  font-size: 26px;
+	  margin-bottom: 20px;
+	}
+
+	#benefits p {
+	  width: 320px;
```

Comment:
You could potentially get away with not even setting `width`s at all here. That way your content would always be 100% wide. Then on larger devices, you could then start to impose `width`s, particularly when you introduce multi column layouts with `float`s and such